test/Faker/Provider/BaseTest.php

```diff
@@ -326,7 +326,8 @@ public function testOptionalReturnsProviderValueWhenCalledWithWeight1()
     {
         $faker = new \Faker\Generator();
         $faker->addProvider(new \Faker\Provider\Base($faker));
-        $this->assertNotNull($faker->optional(1)->randomDigit);
+        // BC: $this->assertNotNull($faker->optional(1)->randomDigit);
```

Comment:
remove it completely, commented out code is a debt. Same for the 2 other occurrences.